queue_job/controllers/main.py

```diff
@@ -122,6 +122,8 @@ def retry_postpone(job, message, seconds=None):
             job.store()
             env.cr.commit()
 
+        # FIXME: this exception never triggers up, so it never reaches
```

Comment:
reason?

Reply:
If I remember correctly, there was other part of code that would catch an exception and then it would go into infinite retries (not knowing when to stop).